spacemacs/compile-comments.vim

```diff
@@ -7,20 +7,20 @@ nnoremap <leader>cc    :action CompileDirty<CR>
 vnoremap <leader>cc    :action CompileDirty<CR>
 
 " Comment lines
-nnoremap <leader>cl    :action CommentByLineComment<CR>:action VimVisualExitMode<CR>
-vnoremap <leader>cl    :action CommentByLineComment<CR>:action VimVisualExitMode<CR>
+nnoremap <leader>cl    :action CommentByLineComment<CR>
+vnoremap <leader>cl    :action CommentByLineComment<CR><Esc>
```

Comment:
All the replacement of `VimVisualExitMode` with `<Esc>` do not work as expected. For example with `SPC c l`, if I select more than one line of code, only 1 of them is commented.

If you are sure that `VimVisualExitMode` will never come back again, then just remove all `<Esc>`.
Even if the user remains in visual mode, at least the comment actions work as expected.

When the issue that you linked will be solved, we can add `<Esc>` back again :)